TRENZ.Docs.API/Services/MarkdownFileProcessingService.cs

```diff
@@ -13,13 +14,15 @@ namespace TRENZ.Docs.API.Services;
 
 public class MarkdownFileProcessingService : IFileProcessingService
 {
+    private static string ToMd5(string text) => Convert.ToHexString(MD5.HashData(Encoding.UTF8.GetBytes(text)));
```

Comment:
Consider xxHash.

Reply:
Would it be that beneficial to speed up this method? I'd say leave it as it is